index.html

```diff
@@ -0,0 +1,130 @@
+      </p> -->
+      <img src="assets/images/intro2.png" alt="">
+      <p id="description">
+        The Challenge is open to all Fintechs on the African continent. Interested Fintechs can apply by sending details of their product as well as a demo  <a href="http://" target="_blank" rel="noopener noreferrer">here</a>. These products must address one or more of a specific set of pain points which can be found <a href="http://" target="_blank" rel="noopener noreferrer">here</a>.
+      </p>
+      <p id="when">
+        2022.11.09(WED) ~ 12(SAT)
+      </p>
+      <p id="where">
+        Salle Evala, Hotel 2 Fevrier, Lomé - TOGO
+      </p>
+    </section>
+    <section id="main-program">
+      <h1>Main Program</h1>
+      <span class="program-bar"></span>
+      <ul class="program">
+        <li class="item">
+          <img src="assets/images/program_icon_01.png" alt="Program 1">
+          <p id="title">Lecture</p>
+          <p id="description">
+            Listen to thes speakers from various countries about the message of sharing and opening
+          </p>
+        </li>
+        <li class="item">
+          <img src="assets/images/program_icon_01.png" alt="Program 1">
+          <p id="title">Workshops</p>
+          <p id="description">
+            Listen to thes speakers from various countries about the message of sharing and opening
+          </p>
+        </li>
+        <li class="item">
+          <img src="assets/images/program_icon_01.png" alt="Program 1">
+          <p id="title">Demo</p>
+          <p id="description">
+            Listen to thes speakers from various countries about the message of sharing and opening
+          </p>
+        </li>
+
+        <li class="item">
+          <img src="assets/images/program_icon_01.png" alt="Program 1">
+          <p id="title">Conference</p>
+          <p id="description">
+            Listen to thes speakers from various countries about the message of sharing and opening
+          </p>
+        </li>
+        <li class="item">
+          <img src="assets/images/program_icon_01.png" alt="Program 1">
+          <p id="title">Livestream</p>
+          <p id="description">
+            Listen to thes speakers from various countries about the message of sharing and opening
+          </p>
+        </li>
+      </ul>
+      <a id="join-program" href="#contact-form"><button class="btn">Join CC Campaign</button></a>
+      <a id="see-more-program" href="#contact-form">SEE THE WHOLE PROGRAM</a>
+    </section>
+    <section id="speakers">
+    </section>
+    <section id="partner">
+      <h1>Partner</h1>
+      <span class="partner-bar"></span>
+      <ul>
+        <li><img src="assets/images/sonsors/iamport.png" alt=""></li>
+        <li><img src="assets/images/sonsors/kakao.png" alt=""></li>
+        <li><img src="assets/images/sonsors/line-friends.png" alt=""></li>
+        <li><img src="assets/images/sonsors/mozilla.png" alt=""></li>
+      </ul>
+
```

Comment:
Kindly consider making the partners' logos a bit smaller such as not to overshadow their heading. Cindy's design can give a clue to that.

Your design:
![image](https://user-images.githubusercontent.com/20554583/198849216-071e6cb6-a340-443e-93f6-b019d825ec9b.png)

Cindy's design:
![image](https://user-images.githubusercontent.com/20554583/198849233-53e00fa9-f305-4f0f-84f7-9cc58065206b.png)

Reply:
@emmiiorji, please, I don't get you. Partner logo or main program logo ?

Reply:
Hi @amentetteh

Sorry, that was the incorrect screenshot. I've updated it.

Reply:
Ok

Reply:
- [ ] As required be the previous reviewer kindly make the logos for the partner section smaller to match the Figma design as much as possible 🔥🙏🏻😊💪🏻